examples/partitioned_indexes.py

```diff
@@ -0,0 +1,52 @@
+"""
+Given a partitioned dataset of embeddings, create an index per partition
+"""
+
+from autofaiss import build_partitioned_indexes
+from pyspark.sql import SparkSession  # pylint: disable=import-outside-toplevel
+
+from pyspark import SparkConf, SparkContext
+
+
+def create_spark_session():
+    # PEX file packaging your Python environment and accessible on yarn by all executors
+    os.environ["PYSPARK_PYTHON"] = "/home/ubuntu/autofaiss.pex"
+    spark = (
+        SparkSession.builder.config("spark.submit.deployMode", "client")
+        .config("spark.executorEnv.PEX_ROOT", "./.pex")
+        .config("spark.task.cpus", "16")
```

Comment:
Maybe you should overide executor-cores to and explain that faiss needs a lot of CPU?